include/options/option/business/browser.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * @package Polylang
+ */
+
+namespace WP_Syntex\Polylang\Options\Option\Business;
+
+use WP_Syntex\Polylang\Options\Option\Primitive\Boolean;
+use WP_Syntex\Polylang\Options\Options;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class defining the "Detect browser language" boolean option.
+ * /!\ Sanitization depends on `force_lang`: this option must be set AFTER `force_lang`.
+ *
+ * @since 3.7
+ *
+ * @phpstan-import-type Schema from \WP_Syntex\Polylang\Options\Option\Abstract_Option
+ */
+class Browser extends Boolean {
+	/**
+	 * Sanitizes option's value.
+	 * Can populate the `$errors` property with blocking and non-blocking errors: in case of non-blocking errors,
+	 * the value is sanitized and can be stored.
+	 *
+	 * @since 3.7
+	 *
+	 * @param bool    $value   Value to sanitize.
+	 * @param Options $options All options.
+	 * @return bool The sanitized value. The previous value in case of blocking error.
+	 */
+	protected function sanitize( $value, Options $options ) {
+		if ( 3 === $options->get( 'force_lang' ) && ! class_exists( 'PLL_Xdata_Domain', true ) ) {
+			// Cannot share cookies between domains without Polylang Pro.
+			return false;
+		}
+
+		$value = parent::sanitize( $value, $options );
+
+		/** @var bool $value */
+		if ( $this->has_blocking_errors() ) {
+			// Blocking error.
+			return $value;
+		}
+
+		return $value;
```

Comment:
Sorry, I don't understand.
```suggestion
		return parent::sanitize( $value, $options );
```

Reply:
Done in [26c62f1](https://github.com/polylang/polylang/pull/1451/commits/26c62f19156bbd7ec8daba5ccd77cd7da79382c8).